app/models/emoji.rb

```diff
@@ -190,6 +190,7 @@ def self.load_custom
             e.name = emoji.name
             e.url = emoji.upload&.url
             e.group = emoji.group || DEFAULT_GROUP
+            e.created_by = User.find(emoji.user_id).username # not sure if we want to do this same thing below when the skip_db global setting is on?
```

Comment:
```suggestion
            e.created_by = User.where(id: emoji.user_id).pick(:username) # not sure if we want to do this same thing below when the skip_db global setting is on?
```

A bit more efficient because it doesn't allocate a full blown `User` object.